tests/sentry/api/endpoints/test_organization_member_index.py

```diff
@@ -498,49 +498,56 @@ def test_cannot_invite_retired_role_with_flag(self):
 class OrganizationMemberPermissionRoleTest(OrganizationMemberListTestBase, HybridCloudTestMixin):
     method = "post"
 
-    def test_manager_invites(self):
-        manager_user = self.create_user("manager@localhost")
-        self.manager = self.create_member(
-            user=manager_user, organization=self.organization, role="manager"
-        )
-        self.login_as(user=manager_user)
-
-        data = {"email": "eric1@localhost", "role": "owner", "teams": [self.team.slug]}
-        self.get_error_response(self.organization.slug, **data, status_code=400)
+    def invite_all_helper(self, role):
+        roles = ["owner", "manager", "member"]
```

Comment:
nit: I think it could be good to specify this variable to not get it confused with the `role` arg, maybe something like `invite_roles`.
Oh you also called them `invite_role` individually in the loop lol